Preparing release v1.5.0

david-terrett opened the pull request:
This release updates EFRA to SOFA 2018-01-30

timj commented on README.rst:
```diff
@@ -34,6 +34,11 @@ Bugfixes
+* ERFA 1.5.0 and SOFA "20180130"
+
+  + There are no differences between ERFA 1.4.0 and SOFA "20180130" except
+    for the ``eraVersion`` anderaSofaVersion`` functions added in ERFA 1.4.0.
```
Typo here with "anderaSofaVersion" and don't we mean v1.5.0?

timj commented on RELEASE.rst:
```diff
@@ -22,7 +22,8 @@ Instructions for releasing ERFA
 * Update the version info of the shared library in the `ERFA_LIB_VERSION_INFO`
   macro of `configure.ac`. Follow the instructions in `Version numbering` below.
-  Also be sure to the ``OFA_VERSION` macro in the `configure.ac`.
+  Also be sure to the update ``SOFA_VERSION`` macro in ``configure.ac`` if
```
Extra "the"

david-terrett commented:
Tim,

I fixed those typos and updated my repository and I think I have created a new pull request (or updated the existing one – I’m not quite sure).  Do you see the updated request? I’m not convinced I’ve done it right.

David

From: Tim Jenness <notifications@github.com>
Sent: 19 July 2019 18:20
To: liberfa/erfa <erfa@noreply.github.com>
Cc: Terrett, David (STFC,RAL,RALSP) <david.terrett@stfc.ac.uk>; Author <author@noreply.github.com>
Subject: Re: [liberfa/erfa] Preparing release v1.5.0 (#51)


@timj commented on this pull request.

________________________________

In README.rst<https://github.com/liberfa/erfa/pull/51#discussion_r305449589>:

> @@ -34,6 +34,11 @@ Bugfixes

 ERFA includes smaller changes that may or may not eventually make it into SOFA,

 addressing localized bugs or similar smaller issues:



+* ERFA 1.5.0 and SOFA "20180130"

+

+  + There are no differences between ERFA 1.4.0 and SOFA "20180130" except

+    for the ``eraVersion`` anderaSofaVersion`` functions added in ERFA 1.4.0.

Typo here with "anderaSofaVersion" and don't we mean v1.5.0?

________________________________

In RELEASE.rst<https://github.com/liberfa/erfa/pull/51#discussion_r305449655>:

> @@ -22,7 +22,8 @@ Instructions for releasing ERFA



 * Update the version info of the shared library in the `ERFA_LIB_VERSION_INFO`

   macro of `configure.ac`. Follow the instructions in `Version numbering` below.

-  Also be sure to the ``OFA_VERSION` macro in the `configure.ac`.

+  Also be sure to the update ``SOFA_VERSION`` macro in ``configure.ac`` if

Extra "the"

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/liberfa/erfa/pull/51?email_source=notifications&email_token=ABVYCMB4BOWUIWWXT3A55T3QAHZVTA5CNFSM4IFHS46KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB7AWUBQ#pullrequestreview-264333830>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABVYCMATKBMPDYV2K23J5LDQAHZVTANCNFSM4IFHS46A>.

timj commented:
You did update the pull request. So long as you keep pushing to the same branch name GitHub will handle it fine (you can force push to your branch as well if you need to edit commits or rebase). One oddity is the unnecessary merge branch that has turned up in the branch. Those can happen if your local branch and the remote branch are out of sync and you do a `git pull` on it.

david-terrett commented:
Erik,

It is probably best if you do it. I’m not that familiar with git and if I do it there is a more than sporting chance that I mess it up which would mean even more work for you to straighten things out.

David

From: Erik Tollerud <notifications@github.com>
Sent: 22 July 2019 20:13
To: liberfa/erfa <erfa@noreply.github.com>
Cc: Terrett, David (STFC,RAL,RALSP) <david.terrett@stfc.ac.uk>; Mention <mention@noreply.github.com>
Subject: Re: [liberfa/erfa] Preparing release v1.5.0 (#51)


@eteq requested changes on this pull request.

The code here all looks good, thanks @david-terrett<https://github.com/david-terrett>! (and @timj<https://github.com/timj> for reviewing before I got to it!)

One minor issue: one of the commits has the changelog "Preparing release v1.5.0". Quite understandably, you followed the instructions in the RELEASE.rst to do that, but in general I try to reserve that changelog entry for the actual release commit (and at the very least there will be the merge commit when this PR is merged).

So can you change your commit message to some other text like "updated to sofa 20180130"? I can then make one additional commit directly into master with the "Preparing release v1.5.0" commit message (in which I will update the release instructions to reflect this discussion!) . Or if you don't know how/don't want to I can do that myself. (I try not to force push to others' branches without them consenting.)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/liberfa/erfa/pull/51?email_source=notifications&email_token=ABVYCMDHNWZXRKWW5I2Y6QLQAYBB5A5CNFSM4IFHS46KYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB7FZD5I#pullrequestreview-264999413>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ABVYCMDNBJFEAGL5DISJ6BTQAYBB5ANCNFSM4IFHS46A>.

eteq commented:
Alright, this is looking good now so I'm going to merge and then I'll do a new release momentarily.  Thanks @david-terrett for your contributions here!
